doc/createrepo_c.8

```diff
@@ -151,7 +151,7 @@ Which compression type to use.
 Which compression type to use (even for primary, filelists and other xml).
 .SS \-\-keep\-all\-metadata
 .sp
-Keep groupfile and updateinfo from source repo during update.
+Keep all additional metadata (not primary, filelists and other xmls, neither theirs compressed variants) from source repo during update.
```

Comment:
`neither theirs` -> `nor their`

Also, don't forget to fix the information in the code, since `createrepo_c --help` has to agree too.

Reply:
Updated, thank you. :+1: